core/src/test/java/org/apache/struts2/dispatcher/PrepareOperationsTest.java

```diff
@@ -73,6 +73,7 @@ public void testWrappedRequestCleanup() {
         });
         IntStream.range(0, mockedRecursions - 1).forEach(i -> prepare.cleanupWrappedRequest(req));
 
+        // Assert org.apache.struts2.dispatcher.Dispatcher#cleanUpRequest has not yet run
```

Comment:
?

Reply:
This comment was meant to clarify the intent of line 77. This unit test is testing that if prepare#wrapRequest is run X times, then prepare#cleanupWrappedRequest must also be run X times before the request is cleaned up. We do not want to clean up too early (line 77) and we don't want to forget to clean up (line 79).